oauth2l/__init__.py

```diff
@@ -393,6 +417,8 @@ def main(argv=None):
     try:
         exit_code = args.func(args)
     except BaseException as e:
+        import traceback
```

Comment:
Two things:

1. Move the import to the top.
2. We probably want to hide this behind a flag, since the traceback is rarely useful for end-users.